src/components/Filters.js

```diff
@@ -99,47 +99,48 @@ class Filters extends React.Component {
+                        <div className="kernel-row">
+                            <div>{this.state.filter[0]}</div>
+                            <div>{this.state.filter[1]}</div>
+                            <div>{this.state.filter[2]}</div>
+                        </div>
+                        <div className="kernel-row">
+                            <div>{this.state.filter[3]}</div>
+                            <div>{this.state.filter[4]}</div>
+                            <div>{this.state.filter[5]}</div>
+                        </div>
+                        <div className="kernel-row">
+                            <div>{this.state.filter[6]}</div>
+                            <div>{this.state.filter[7]}</div>
+                            <div>{this.state.filter[8]}</div>
                         </div>
-                        <form>
-                            <label>
-                                Choose a filter:
-                                <select value={this.state.value} onChange={this.handleChange}>
-                                    <option value="identity">Identity</option>
-                                    <option value="blur">Blur</option>
-                                    <option value="bottom sobel">Bottom sobel</option>
-                                    <option value="emboss">Emboss</option>
-                                    <option value="left sobel">Left sobel</option>
-                                    <option value="outline">Outline</option>
-                                    <option value="right sobel">Right sobel</option>
-                                    <option value="sharpen">Sharpen</option>
-                                    <option value="top sobel">Top sobel</option>
-                                </select>
-                            </label>
-                        </form>
-
                     </div>
-                    <img id="filter-applied" src={this.state.src} alt="amsterdam with filter applied" width="500px"></img>
+                    <form>
+                        <label>
+                            Choose a filter:
+                            <select value={this.state.value} onChange={this.handleChange}>
+                                <option value="identity">Identity</option>
+                                <option value="blur">Blur</option>
+                                <option value="bottom sobel">Bottom sobel</option>
+                                <option value="emboss">Emboss</option>
+                                <option value="left sobel">Left sobel</option>
+                                <option value="outline">Outline</option>
+                                <option value="right sobel">Right sobel</option>
+                                <option value="sharpen">Sharpen</option>
+                                <option value="top sobel">Top sobel</option>
+                            </select>
+                        </label>
```

Comment:
Also disappointed that I didn't catch this myself earlier, but AFAIK the `<label>` and `<select>` are neighbour tags. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select